docs/docs/APIdocs/Layers/Convolution-Layers/SpatialFullConvolution.md

```diff
@@ -2,11 +2,11 @@
 
 **Scala:**
 ```scala
-val m  = SpatialFullConvolution[Tensor[T], T](nInputPlane, nOutputPlane, kW, kH, dW=1, dH=1, padW=0, padH=0, adjW=0, adjH=0,nGroup=1, noBias=false,wRegularizer=null,bRegularizer=null)
+val m  = SpatialFullConvolution[T](nInputPlane, nOutputPlane, kW, kH, dW=1, dH=1, padW=0, padH=0, adjW=0, adjH=0,nGroup=1, noBias=false,wRegularizer=null,bRegularizer=null)
 ```
 or
 ```scala
-val m = SpatialFullConvolution[Table, T](InputPlane, nOutputPlane, kW, kH, dW=1, dH=1, padW=0, padH=0, adjW=0, adjH=0,nGroup=1, noBias=false,wRegularizer=null,bRegularizer=null)
+val m = SpatialFullConvolution[T](InputPlane, nOutputPlane, kW, kH, dW=1, dH=1, padW=0, padH=0, adjW=0, adjH=0,nGroup=1, noBias=false,wRegularizer=null,bRegularizer=null)
```

Comment:
But maybe it's better to make users know the layer support both tensor and table input, and show how to use them